src/modules/85-cv/pages/slos/CVSLOListingPage.utils.ts

```diff
@@ -477,6 +478,7 @@ export const getSLODashboardWidgetsParams = (
       userJourneyIdentifiers: getFilterValueForSLODashboardParams(getString, filterState.userJourney),
       targetTypes: getFilterValueForSLODashboardParams(getString, filterState.targetTypes) as TargetTypesParams[],
       sliTypes: getFilterValueForSLODashboardParams(getString, filterState.sliTypes) as SLITypesParams[],
+      filter: search,
```

Comment:
As search is an optional parameter , it might be undefined . So can we add default value as well 

```suggestion
      filter: defaultTo(search, '')
```

Reply:
search is a state variable whose value has been already initialised as empty string.